x-pack/plugins/observability_solution/slo/public/pages/slos/components/card_view/slo_card_item.tsx

```diff
@@ -238,8 +232,12 @@ export function SloCardChart({
   return (
     <Chart>
       <Settings
-        // TODO connect to charts.theme service see src/plugins/charts/public/services/theme/README.md
-        baseTheme={LEGACY_DARK_THEME}
+        baseTheme={charts.theme.useChartsBaseTheme()}
```

Comment:
This will handle both dark and light theme, right?

Reply:
this seems to be the recommended way so it should